Subscriber/AdditionalAccessTokenClaimsAndHeaderSubscriber.php

```diff
@@ -29,14 +29,16 @@ public static function getSubscribedEvents(): array
 
     public function addClaims(JWTCreatedEvent $event): void
     {
+        $now = new \DateTimeImmutable();
```

Comment:
You should use `symfony/clock` instead like here : https://github.com/lexik/LexikJWTAuthenticationBundle/blob/3.x/Services/JWSProvider/LcobucciJWSProvider.php#L46-L50